src/PhpSpreadsheet/Writer/Xlsx/Drawing.php

```diff
@@ -179,6 +185,15 @@ public function writeDrawing(XMLWriter $objWriter, BaseDrawing $pDrawing, $pRela
             $objWriter->writeAttribute('id', $pRelationId);
             $objWriter->writeAttribute('name', $pDrawing->getName());
             $objWriter->writeAttribute('descr', $pDrawing->getDescription());
+
+            //a:hlinkClick
+            if ($pHlinkClickId >= 0) {
```

Comment:
This `if` increases the Condition count from 6 to 7, which raised the rating from B to C.
The project's current configuration of Scrutinizer forbids ratings above or equal to C.

Reply:
[Scrutinizer about Drawing::writeDrawings()](https://scrutinizer-ci.com/g/PHPOffice/PhpSpreadsheet/inspections/4adc3239-9aac-4b9f-9d9f-96ff77f47983/code-structure/operation/PhpOffice%5CPhpSpreadsheet%5CWriter%5CXlsx%5CDrawing%3A%3AwriteDrawings)

Reply:
Indeed, the `if` should be extract to new private method.